account_invoice_validate_send_email/models/account_move.py

```diff
@@ -53,6 +57,14 @@ def action_post(self):
                 continue
             if not move.send_invoice or move.invoice_sent:
                 continue
+            # Skip sending invoice if the invoice is not physical and the customer is B2C.
+            if (
+                move.partner_id.user_type == "b2c"
```

Comment:
To be slightly safer.
```suggestion
                move.commercial_partner_id.user_type == "b2c"
```